python/lsst/ts/externalscripts/coordination/script.py

```diff
@@ -0,0 +1,219 @@
+__all__ = ["LaserCoordination"]
```

Comment:
Please, rename file to reflect what is inside. I suggest `laser_coordination.py`